hudi-common/src/main/java/org/apache/hudi/common/table/view/PriorityBasedFileSystemView.java

```diff
@@ -25,6 +25,7 @@
 import org.apache.hudi.common.model.HoodieFileGroupId;
 import org.apache.hudi.common.table.timeline.HoodieInstant;
 import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.util.Functions;
 import org.apache.hudi.common.util.Functions.Function0;
```

Comment:
import Function0, follow the existing style

Reply:
done